superset-frontend/plugins/plugin-chart-echarts/src/Pie/transformProps.ts

```diff
@@ -248,6 +311,18 @@ export default function transformProps(
       ...getLegendProps(legendType, legendOrientation, showLegend),
       data: keys,
     },
+    graphic: showTotal
+      ? {
+          type: 'text',
+          ...getTotalValuePadding({ chartPadding, donut, width, height }),
+          style: {
+            text: t(`Total: ${numberFormatter(totalValue)}`),
```

Comment:
Late to the party, but translated strings need to be constant. This needs to be something like:
```js
t('Total: %s', numberFormatter(totalValue))
```